src/components/diagrams/singleLineDiagram/single-line-diagram-content.js

```diff
@@ -373,6 +373,13 @@ const SingleLineDiagramContent = forwardRef((props, ref) => {
         diagramSizeSetter,
     ]);
 
+    const handleWarning = useCallback(
+        (id, svgType) => {
+            setWarning(id, svgType);
```

Comment:
- The first parameter "id" is not needed, we directly have access to props.diagramId
- It would be cleaner to define the error message here instead of in diagram-pane.js::643

So after those changes we could call `setWarning(props.diagramId, DiagramType.VOLTAGE_LEVEL === svgType ? 'VoltageLevelNotFound' : 'SubstationNotFound');`

Reply:
done